modules/api-svc/src/lib/model/InboundTransfersModel.js

```diff
@@ -77,7 +75,9 @@ class InboundTransfersModel {
     updateStateWithError(err) {
         this.data.lastError = err;
         this.data.currentState = SDKStateEnum.ERROR_OCCURRED;
-        return this._save();
+        return this.data.transferId // if no transferId - we are in fxQuote flow
```

Comment:
Why do we need this condition?

Reply:
to save state either for _quote_ or for _fxQuote_